src/infrastructure/mongo/persistence/crowdaction.persistence.ts

```diff
@@ -91,5 +91,8 @@ export class CrowdActionPersistence implements Omit<ICrowdAction, 'id' | 'create
 
     @Prop({ type: [CrowdActionBadgePersistenceSchema], required: false, array: true })
     readonly badges: IBadge[];
+
+    @Prop({ type: Object, required: true })
```

Comment:
Also can you make a schema for BadgeConfig? Just so this isn't a generic object.